spark/src/test/scala/com/alibaba/graphar/GraphAr2Neo4j.scala

```diff
@@ -0,0 +1,131 @@
+/** Copyright 2022 Alibaba Group Holding Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.graphar
+
+import com.alibaba.graphar.datasources._
+import com.alibaba.graphar.reader.{VertexReader, EdgeReader}
+
+import java.io.{File, FileInputStream}
+import org.yaml.snakeyaml.Yaml
+import org.yaml.snakeyaml.constructor.Constructor
+import scala.beans.BeanProperty
+import org.apache.spark.sql.{DataFrame, SaveMode, SparkSession}
+import org.apache.hadoop.fs.{Path, FileSystem}
+
+class GraphAr2Neo4jExample {
+  // connect to the Neo4j instance
+  val spark = SparkSession.builder()
+    .config("neo4j.url", "bolt://localhost:7687")
+    .config("neo4j.authentication.type", "basic")
+    .config("neo4j.authentication.basic.username", sys.env.get("Neo4j_USR").get)
+    .config("neo4j.authentication.basic.password", sys.env.get("Neo4j_PWD").get)
+    .config("spark.master", "local")
+    .getOrCreate()
+
+  // read Person vertices from GraphAr and write to Neo4j
+  def testWriteVerticesToNeo4j(): Unit = {
+    // read vertex info yaml
+    val vertex_yaml = getClass.getClassLoader.getResource("gar-test/neo4j/person.vertex.yml").getPath
```

Comment:
I do not find the chunk file of neo4j in gar-test repo. Do we need to put one copy of these file to gar-test?

Reply:
Thanks, the chunk files are added into gar-test now.